layouts/shortcodes/fields/tech.html

```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  <label class="label">Are you actively in the tech industry?*</label>
```

Comment:
I would reword as "Are you currently a professional in the tech industry or interested in becoming one?"

Reply:
updated

Reply:
doesnt look like this was reworded

Reply:
the aria labels were but not label